src/main/java/uk/gov/companieshouse/web/accounts/model/smallfull/notes/fixedassetsinvestments/FixedAssetsInvestments.java

```diff
@@ -10,7 +11,8 @@
 @ValidationModel
 public class FixedAssetsInvestments {
 
-    @ValidationMapping("$.fixed_asset_investments.details")
+    @NotBlank(message = "{fixedAssetsInvestments.details.missing}")
```

Comment:
This isn't required since the api validation enforces this. This can be removed, along with the corresponding ValidationMessage property.